app/static/ie-status.json

```diff
@@ -2934,7 +2934,7 @@
     "name": "Number Built-ins (ES6)",
     "category": "JavaScript",
     "summary": "New methods and properties on Number, including isFinite, isInteger, isSafeInteger, isNaN, EPSILON, MIN_SAFE_INTEGER, and MAX_SAFE_INTEGER.",
-    "link": "https://people.mozilla.org/~jorendorff/es6-draft.html",
+    "link": "http://ecma-international.org/ecma-262/6.0/index.html",
```

Comment:
Perhaps in some cases, but really these features span many spec sections so it's hard to link to a specific one. I'd rather handle this as part of a separate PR.